src/utils/DarCollectionUtils.js

```diff
@@ -212,6 +209,42 @@ export const extractUserRPVotesFromBucket = (bucket, user, isChair) => {
   )(votes);
 };
 
+export const getMatchDataForBuckets = async (buckets) => {
+  const electionIdBucketMap = {}; //{ referenceId: bucket} }
```

Comment:
This is the kind of thing where I think TypeScript would help us clean up the typing.